locale.c

```diff
@@ -6141,10 +6141,106 @@ S_langinfo_sv_i(pTHX_
 
         const char * retval = nl_langinfo(item);
         Size_t total_len = strlen(retval);
+        char separator;
+
+        if (UNLIKELY(item == ALT_DIGITS) && total_len > 0) {
+
+            /* The format of the return from nl_langinfo(ALT_DIGITS) is not
+             * specified by the POSIX Standard, nor much by vendors' man pages.
+             * (although the Standard does specify how they should be formatted
+             * internally in <langinfo.h>, and a conforming implementation
+             * would likely try to do the least work possible in transforming
+             * them for output).  There are up to 100 different values, the
```

Comment:
The section of that page I was referring to is headed "LC_TIME C-Language Access".

The internal format is described further up and is different from the format documented under  "LC_TIME C-Language Access".

The internal format would be documented under "LC_TIME Locale Definition" is:
```
"0";"1";...
```
while the value documented under "LC_TIME C-Language Access" is:
```
0;1;...
```
though burying it in this page is misleading 😒.